ui/src/components/Pages/TrafficPage/TrafficPage.tsx

```diff
@@ -122,6 +122,7 @@ export const TrafficPage: React.FC<TrafficPageProps> = ({setAnalyzeStatus}) => {
         ws.current.onopen = () => {
             setWsConnection(WsConnectionStatus.Connected);
             ws.current.send(query);
+            ws.current.send("0");
```

Comment:
can we change it in a way that it will don't need to send the "0" here?
I understand it is disabling the full data streaming but I think we can assume it is disabled until someone enabled it..

(maybe I misunderstand and it is not what the "0" here for

Reply:
With https://github.com/up9inc/mizu/pull/819/commits/df4405eedd399abc7ac50b72ebff5bcfce610f12 I have made it empty string. We need to send message to the server regardless of the value.

Reply:
@mertyildiran 
why do we need to send this message?
can't we initilalize the value of  enableFullEntries to false (which is the default of boolena but maybe better to be specifically written) and then enabled by user action (and if needed it will send message of disable) but I don't see reason to send this 0.

am I miss anything?

Reply:
@gadotroee it's because in the second iteration of this loop:

https://github.com/up9inc/mizu/blob/df4405eedd399abc7ac50b72ebff5bcfce610f12/agent/pkg/api/socket_routes.go#L118-L130

this line is blocking until seeing a message:

https://github.com/up9inc/mizu/blob/df4405eedd399abc7ac50b72ebff5bcfce610f12/agent/pkg/api/socket_routes.go#L119

So there should be always two messages from client to server.